packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -92,22 +92,22 @@ export class ComponentPage extends React.Component<IComponentPageProps, {}> {
     if (bestPractices && dos && donts) {
       links.push(
         <div className='ComponentPage-navLink' key='nav-link'>
-          <Link href={ this._baseUrl + '#Best Practices' }>Best Practices</Link>
+          <Link { ...{ href: this._baseUrl + '#Best Practices' } }>Best Practices</Link>
```

Comment:
It's because example-app-base refers to the previous version of fabric (from npm) due to the circular dependency, and thus, the old link component is not compatible with the updated react typings. So as a temporary workaround until we're fully on the new typings, we have to work around it like this.